### ISSUE TITLE
No package python-zmq available when bootstrapping using via git

### ISSUE THREAD

**Issue by hubez** [saltstack/salt-bootstrap#824]
### Description of Issue/Question

Installing salt from git does not install the salt yum repo correctly such that package python-zmq cannot be found, and thus salt will not run.
### Setup

Fresh EC2 instance using Amazon Linux
### Steps to Reproduce Issue

$ bootstrap-salt.sh -X -G git v2015.8.8

You will eventually see the "No package python-zmq available" error:

```
/tmp/install_salt.sh: line 3712: STABLE_REV: unbound variable
Loaded plugins: priorities, update-motd, upgrade-helper
957 packages excluded due to repository priority protections
Package python26-PyYAML-3.10-3.10.amzn1.x86_64 already installed and latest version
Package python26-m2crypto-0.21.1-15.14.amzn1.x86_64 already installed and latest version
Package python26-crypto-2.6.1-1.12.amzn1.x86_64 already installed and latest version
Package python-msgpack-0.4.6-1.el6.x86_64 already installed and latest version
No package python-zmq available.
```
### Versions and Systems

```
bootstrap-salt.sh -v
bootstrap-salt.sh -- Version 2016.04.18
```

```
uname -a
Linux ip-10-128-71-208 4.1.10-17.31.amzn1.x86_64 #1 SMP Sat Oct 24 01:31:37 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux
```

**Comment by jfindlay** [comment]
@hubez, thanks for reporting.  Is it possible for you to retry with the [latest release](https://raw.githubusercontent.com/saltstack/salt-bootstrap/v2016.05.11/bootstrap-salt.sh)?  If not it is fine.

**Comment by rallytime** [comment]
@hubez This should be fixed once #906 is merged into develop. (We'll be doing a new stable release of bootstrap in the next week or so.) I just successfully bootstrapped a fresh Amazon Linux box on EC2 with the version of the script present in #906 with `sh install_salt.sh -P git v2015.8.10`:

```
Writing /usr/lib/python2.6/site-packages/salt-2015.8.10-py2.6.egg-info
 *  INFO: Running install_amazon_linux_ami_git_post()
 *  INFO: Running install_amazon_linux_ami_restart_daemons()
Starting salt-minion daemon:                               [  OK  ]
 *  INFO: Running daemons_running()
 *  INFO: Salt installed!
[root@ip-10-155-10-224 ~]# salt-call --versions
Salt Version:
           Salt: 2015.8.10

Dependency Versions:
         Jinja2: 2.7.3
       M2Crypto: 0.21.1
           Mako: Not Installed
         PyYAML: 3.11
          PyZMQ: 14.5.0
         Python: 2.6.9 (unknown, Dec 17 2015, 01:08:55)
           RAET: Not Installed
        Tornado: 4.2.1
            ZMQ: 4.0.5
           cffi: Not Installed
       cherrypy: Not Installed
       dateutil: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
        libgit2: Not Installed
        libnacl: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.4.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
         pygit2: Not Installed
   python-gnupg: Not Installed
          smmap: Not Installed
        timelib: Not Installed

System Versions:
           dist:
        machine: x86_64
        release: 4.4.11-23.53.amzn1.x86_64
```

I think with that information this issue can likely be closed. @hubez can you confirm?

**Comment by rallytime** [comment]
Closing since #906 has been merged now.
